ds3-autogen-c/src/main/java/com/spectralogic/ds3autogen/c/models/Parameter.java

```diff
@@ -61,4 +61,19 @@ public String toString() {
         }
         return parameterType + parameterPointerType + " " + name;
     }
+
+    public boolean equals(final Object obj) {
+        if (!(obj instanceof Parameter)) {
+            return false;
+        }
+        return getName().equals(((Parameter) obj).getName())
+            && getParameterType().equals(((Parameter) obj).getParameterType())
+            && getParameterPointerType().equals(((Parameter) obj).getParameterPointerType())
+            && getTypeModifier().equals(((Parameter) obj).getTypeModifier());
+    }
+
+    public int hashCode() {
+        return name.hashCode() ^ parameterType.hashCode();
```

Comment:
For consistency, this should use the objects hash like in Ds3Type (parser module), such as:
return Objects.hash(getName(), getParameterType())